modules/jupyter-widget/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@deck.gl/jupyter-widget",
+  "version": "0.1.0",
+  "description": "Jupyter widget for rendering deck.gl in a Jupyter notebook",
+  "license": "MIT",
+  "version": "7.0.0-alpha.3",
+  "keywords": [
+    "jupyter",
+    "jupyterlab",
+    "jupyterlab-extension",
+    "widgets"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/uber/deck.gl.git"
+  },
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
+  "files": [
+    "dist",
+    "src",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "scripts": {
+    "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5 dist/esm dist/es6",
+    "build-es6": "BABEL_ENV=es6 babel src --out-dir dist/es6 --source-maps --ignore 'node_modules/'",
+    "build-esm": "BABEL_ENV=esm babel src --out-dir dist/esm --source-maps --ignore 'node_modules/'",
+    "build-es5": "BABEL_ENV=es5 babel src --out-dir dist/es5 --source-maps --ignore 'node_modules/'",
+    "build": "npm run clean && npm run build-es6 && npm run build-esm && npm run build-es5"
+  },
+  "dependencies": {
+    "@deck.gl/json": "7.0.0-beta.2",
+    "@deck.gl/aggregation-layers": "7.0.0-beta.2",
+    "@deck.gl/layers": "7.0.0-beta.2",
+    "@jupyter-widgets/base": "^1.1.10",
+    "@types/mapbox-gl": "^0.51.4",
+    "@deck.gl/core": "7.0.0-beta.2",
+    "mapbox-gl": "^0.53.1"
+  },
+  "jupyterlab": {
```

Comment:
Out of curiosity, how is this field used? Is that documented somewhere?

Reply:
This can be cut, it's a holdover from the Jupyter widget template.